### PR TITLE
feat(unlock-app): "I am going" feature

### PR DESCRIPTION
**Pull request by SVell**
# Description

Add "I'm Going!" feature block in next to the "Check-ins are a breeze" block on Events.Unlock-Protocol.com

# Issues

Fixes #
Refs #14316

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [x] If my code involves visual changes, I am adding applicable screenshots to this thread

## Release Note Draft Snippet

image
image

### CODE REVIEW

**Comment by ccarfi** [comment]
hi @SVell — 

here's what I'm seeing (more thoughts below the image)
Screenshot 2024-07-26 at 7 01 53 AM

when i look at the preview URL in Vercel from the build, i notice three differences:

1) There are only three images in that feature section, instead of four (like is shown in your screenshot upthread)
2) The images are much too large
3) The images are in a different order

Please take a look at this Loom — am I looking at the live Vercel preview at the correct URL?
Here's the Loom: https://www.loom.com/share/3dde5919715f410287bcf1ca5e35df24

(and here's the preview URL I grabbed from the build log above: https://unlock-app-staging-c6wu88618-unlock-protocol.vercel.app/subscription

**Comment by ccarfi** [comment]
I think the EVENTS page looks ok, but I still think the images in the 2x2 layout are too large — they are much larger than, say, the images in the section above them

**Comment by ccarfi** [comment]
it also breaks the CERTIFICATION page for the same reason
https://unlock-app-staging-c6wu88618-unlock-protocol.vercel.app/certification
Screenshot 2024-07-26 at 7 11 50 AM

**Comment by ccarfi** [comment]
i know there was a request to move it to a 2x2 layout and include four items in the EVENTS page, but maybe we regress back to three items and simply replace one of the items on the EVENTS page so it stays consistent with all the other landing pages.

File: unlock-app/src/components/interface/LockTypeLandingPage.tsx [hunk 1]
```diff
@@ -177,12 +177,12 @@ export const LockTypeLandingPage = ({
         {callToAction?.subtitle && (
           <p className="mt-4 text-center md:w-1/2">{callToAction?.subtitle}</p>
         )}
-        <ul className="grid md:grid-cols-[300px_300px_300px] gap-8 my-8">
+        <ul className="grid lg:grid-cols-2 gap-16 my-8 md:w-4/5">
```
Comment (julien51) [inline unlock-app/src/components/interface/LockTypeLandingPage.tsx:180]: Can you make sure this does not break the pages were there are 3 boxes for example?

**Review by julien51 — APPROVED** [review]

**Pull request merged**